x-pack/metricbeat/module/oracle/tablespace/data.go

```diff
@@ -41,13 +41,14 @@ func (m *MetricSet) transform(in *extractedData) (out map[string]mapstr.M) {
 	out = make(map[string]mapstr.M, 0)
 
 	for _, dataFile := range in.dataFiles {
-		m.addDataFileData(&dataFile, out)
+		dataFileCopy := dataFile
+		m.addDataFileData(&dataFileCopy, out)
 	}
 
 	m.addUsedAndFreeSpaceData(in.freeSpace, out)
 	m.addTempFreeSpaceData(in.tempFreeSpace, out)
 
-	return
+	return out
```

Comment:
since we are using named returns, we can skip explicitly returning out.

Reply:
The problem here is that the lint check fails if we keep return statement empty.